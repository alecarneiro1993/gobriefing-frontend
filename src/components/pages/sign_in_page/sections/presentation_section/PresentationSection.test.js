import React from "react";
import { shallow } from "enzyme";

import { PresentationSection } from ".";

describe("PresentationSection", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<PresentationSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
