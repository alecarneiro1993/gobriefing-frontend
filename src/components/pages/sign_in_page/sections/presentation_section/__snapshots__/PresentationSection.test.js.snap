// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PresentationSection renders without crashing 1`] = `
<Styled(Styled(MuiBox))
  src="https://dslv9ilpbe7p1.cloudfront.net/hownJrwXlTVdURCWN3Sj1g_store_header_image"
/>
`;
